src/coll-index.js

```diff
@@ -513,7 +513,7 @@ class CollInfo extends LitElement {
             ${coll.desc}
           </div>`
         : html``}
-      /* Only show description if it exists, otherwise show filename? */
+      <!--  Only show description if it exists, otherwise show filename --> 
```

Comment:
Minor tweak: if we have description show it. If we have `coll.resources`, then don't show the filename as it'll just be `replay.json` and we're already showing that below and its

Reply:
Some maybe add show filename as else condition for coll.resources